src/py42/sdk/queries/fileevents/filters/risk_filter.py

```diff
@@ -0,0 +1,232 @@
+        - :attr:`RiskIndicator.EmailServiceUploads.YAHOO`
+        - :attr:`RiskIndicator.EmailServiceUploads.ZOHO_MAIL`
+        - :attr:`RiskIndicator.RemovableMedia.AIRDROP`
+        - :attr:`RiskIndicator.RemovableMedia.REMOVABLE_MEDIA`
+        - :attr:`RiskIndicator.FileCategories.AUDIO`
+        - :attr:`RiskIndicator.FileCategories.DOCUMENT`
+        - :attr:`RiskIndicator.FileCategories.EXECUTABLE`
+        - :attr:`RiskIndicator.FileCategories.IMAGE`
+        - :attr:`RiskIndicator.FileCategories.PDF`
+        - :attr:`RiskIndicator.FileCategories.PRESENTATION`
+        - :attr:`RiskIndicator.FileCategories.SCRIPT`
+        - :attr:`RiskIndicator.FileCategories.SOURCE_CODE`
+        - :attr:`RiskIndicator.FileCategories.SPREADSHEET`
+        - :attr:`RiskIndicator.FileCategories.VIDEO`
+        - :attr:`RiskIndicator.FileCategories.VIRTUAL_DISK_IMAGE`
+        - :attr:`RiskIndicator.FileCategories.ZIP`
+        - :attr:`RiskIndicator.MessagingServiceUploads.FACEBOOK_MESSENGER`
+        - :attr:`RiskIndicator.MessagingServiceUploads.MICROSOFT_TEAMS`
+        - :attr:`RiskIndicator.MessagingServiceUploads.SLACK`
+        - :attr:`RiskIndicator.MessagingServiceUploads.WHATSAPP`
+        - :attr:`RiskIndicator.Other.OTHER`
+        - :attr:`RiskIndicator.MessagingServiceUploads.UNKNOWN`
+        - :attr:`RiskIndicator.SocialMediaUploads.FACEBOOK`
+        - :attr:`RiskIndicator.SocialMediaUploads.LINKEDIN`
+        - :attr:`RiskIndicator.SocialMediaUploads.REDDIT`
+        - :attr:`RiskIndicator.SocialMediaUploads.TWITTER`
+        - :attr:`RiskIndicator.UserBehavior.FILE_MISMATCH`
+        - :attr:`RiskIndicator.UserBehavior.OFF_HOURS`
+        - :attr:`RiskIndicator.UserBehavior.REMOTE`
+    """
+
+    _term = u"riskIndicatorNames"
+
+    @staticmethod
+    def choices():
+        return (
+            RiskIndicator.CloudDataExposures.choices()
+            + RiskIndicator.CloudStorageUploads.choices()
+            + RiskIndicator.CodeRepositoryUploads.choices()
+            + RiskIndicator.EmaiServiceUploads.choices()
+            + RiskIndicator.ExternalDevices.choices()
+            + RiskIndicator.FileCategories.choices()
+            + RiskIndicator.MessagingServiceUploads.choices()
+            + RiskIndicator.Other.choices()
+            + RiskIndicator.SocialMediaUploads.choices()
+            + RiskIndicator.UserBehavior.choices()
+        )
+
+    class CloudDataExposures(object):
+        PUBLIC_CORPORATE_BOX = "Public link from corporate Box"
+        PUBLIC_CORPORATE_GOOGLE_DRIVE = "Public link from corporate Google Drive"
+        PUBLIC_CORPORATE_ONEDRIVE = "Public link from corporate OneDrive"
+        SENT_CORPORATE_GMAIL = "Sent from corporate Gmail"
+        SHARED_CORPORATE_BOX = "Shared from corporate Box"
+        SHARED_CORPORATE_GOOGLE_DRIVE = "Shared from corporate Google Drive"
+        SHARED_CORPORATE_ONEDRIVE = "Shared from corporate OneDrive"
+
+        @staticmethod
```

Comment:
I wonder why we didn't just make a base Enum class with `classmethod` `all(cls)` that all these classes inherit from (all the filter classes etc). I have used this approach myself and it works the same way as it is now without having to repeat the same method N times.

Reply:
```
class Enum:
  def all(cls):
    return get_attribute_keys_from_class(cls)
```

Reply:
Not every string filter has choices (like file name, file path, username, etc).  Otherwise `choices` would just return nothing for those. 

That's probably why it got done this way originally, but we could still achieve what you're after by creating separate classes for those that do have choice vs those that don't. That's probably a good idea and would remove some cruft, I can create a ticket for it.

Reply:
All that it would be is to create base class that classes with `choices` could inherit from, and then they get the `all()` method for free

Reply:
Yep exactly. Right now there are some `FileEventFilterStringField`s that have choices and some that don't, and we don't want choices to be on the ones that don't, so we cant just add all() to `FileEventFilterStringField`. We would need to create `FileEventFilterStringOptionField` which inherits from `FileEventFilterStringField` and add the method to that, then change everything that has choices to use that class instead.

Reply:
Ah I forgot that there is already a base class in this situation, that does make it a tad more complicated